drafts/2020-05-05-this-week-in-rust.md

```diff
@@ -105,7 +122,6 @@ decision. Express your opinions now.
 
 ### Online
 
-* [Apr 30. Zurich, CH - Rust Zurich - Security in Rust: cargo-crev and cargo-audit](https://www.meetup.com/Rust-Zurich/events/270169298/).
 * [May  4. Auckland, NZ - Rust AKL - Rust Implementation of Ethereum2.0 + Adding WASM Support to a Native Application](https://www.meetup.com/rust-akl/events/266876545/).
```

Comment:
The newsletter is going out on May 5, which will be after this meetup has passed (sadly, it looks really interesting!). Can you remove this one?

Reply:
Removed the Auckland meetup